scripts/setup.py

```diff
@@ -52,22 +53,26 @@ def install_system_dependencies():
     )
 
 
-def setup_postgresql():
+def setup_postgresql(default_user="postgres", default_password="postgres"):
     """Configure PostgreSQL for the project"""
     try:
+        # Allow overriding the default user and password with environment variables
+        postgres_user = os.getenv("POSTGRES_USER", default_user)
```

Comment:
Let's set the defaults to "postgres" and "postgres" for both in the env too

Reply:
Should have resolved with the lasted commit